android-artifacts.gradle

```diff
@@ -0,0 +1,22 @@
+uploadArchives {
+    /* task androidJavadocs(type: Javadoc) {
+       source = android.sourceSets.main.java.sourceFiles
+   }
+
+   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+       classifier = 'javadoc'
+       //basename = artifact_id
+       from androidJavadocs.destinationDir
+   }*/
+
+    task androidSourcesJar(type: Jar) {
+        classifier = 'sources'
+        //basename = artifact_id
+        from android.sourceSets.main.java.sourceFiles
+    }
+
+    artifacts {
+        archives androidSourcesJar
+        //archives androidJavadocsJar
```

Comment:
It means they're not generated as a javadoc jar. The sources still appear
and thus we can link the sources which also contain the comments and
javadoc information.

On Fri, Jan 23, 2015 at 10:38 AM, dylanrjames <notifications@github.com>
wrote:

> In android-artifacts.gradle
> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6#discussion_r23456836>
> :
>
> > +
> > +   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
> > +       classifier = 'javadoc'
> > +       //basename = artifact_id
> > +       from androidJavadocs.destinationDir
> > +   }*/
> > +
> > +    task androidSourcesJar(type: Jar) {
> > +        classifier = 'sources'
> > +        //basename = artifact_id
> > +        from android.sourceSets.main.java.sourceFiles
> > +    }
> > +
> > +    artifacts {
> > +        archives androidSourcesJar
> > +        //archives androidJavadocsJar
>
> Does this mean that the JavaDoc doesn't appear when you pull it in as a
> Gradle dependency like this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6/files#r23456836>.
>



-- 
--

*Andrew Grosner, Android Developer*

Mobile Strategy, Design & Development

www.raizlabs.com | Check my calendar
<https://www.google.com/calendar/embed?src=andrew.grosner@raizlabs.com&ctz=America/New_York&mode=WEEK>
 | @agrosner <http://www.twitter.com/agrosner> | 716-472-5075
[image: Raizlabs]